executor/adapter.go

```diff
@@ -764,8 +767,28 @@ func FormatSQL(sql string, pps variable.PreparedParams) stringutil.StringerFunc
 		if maxQueryLen := atomic.LoadUint64(&cfg.Log.QueryLogMaxLen); uint64(length) > maxQueryLen {
 			sql = fmt.Sprintf("%.*q(len:%d)", maxQueryLen, sql, length)
 		}
-		return QueryReplacer.Replace(sql) + pps.String()
+		replaceSQL := QueryReplacer.Replace(sql)
+		if len(pps) == 0 {
+			return replaceSQL
+		}
+		return formatPreparedSQL(replaceSQL, pps)
```

Comment:
Should we format prepared SQLs before truncating them? How about moving this `formatPreparedSQL` to line 765?

Reply:
I will fix it.